samcli/commands/local/cli_common/invoke_context.py

```diff
@@ -270,6 +271,15 @@ def __enter__(self) -> "InvokeContext":
         if self._containers_initializing_mode == ContainersInitializationMode.EAGER:
             self._initialize_all_functions_containers()
 
+        for func in self._function_provider.get_all():
+            if func.packagetype == ZIP and func.inlinecode:
+                LOG.warning(
+                    "Warning: Inline code found for function %s."
+                    " Invocation of inline code is not supported for sam local commands.",
```

Comment:
let's not reference the binary here directly. as it would be samdev, sam-beta, sam-nightly. You could get the cli context and try to figure out the command name (could be a useful helper function).

Reply:
I agree, we distribute the binaries sam and sam-nightly but hard code it to sam in the print statements. Same goes for CI jobs where we use sam-beta for the unit/integ tests. I can create a separate PR for that helper function and keep this PR clean as this change would require updating the hard coded binary name for the related unit/integration tests for all the commands as well.